include/zephyr/net/ptp.h

```diff
@@ -6,8 +6,8 @@
 
 #pragma once
 
-#include <net/net_if.h>
```

Comment:
For the commit `subsys: net: l2: ethernet: ....`, the commit subject contains unnecessary and extra info and can be shortened to
`net: ptp: Add master & slave`